app/themes/default/views/layouts/default.html.slim

```diff
@@ -38,6 +38,13 @@ html
             li
               = link_to(t('layout.navbar.help'), '#')
             - if user_signed_in?
+              - if current_user.administrator?
+                li
+                  a.dropdown-toggle data-toggle="dropdown" = current_user.name
+                    b.caret
+                  ul.dropdown-menu
+                    li
+                      = link_to(t('layout.navbar.admin_panel'), admin_path)
```

Comment:
@lowjoel  So `can?(:manage, current_tenant)` ? this works for both global admin and instance admin.